publisher_joint_trajectory_planned:
  ros__parameters:

    controller_name: "scaled_joint_trajectory_controller"
    wait_sec_between_publish: 1

    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint

    check_starting_point: true
    starting_point_limits:
      # shoulder_pan_joint: [-0.1,0.1]
      # shoulder_lift_joint: [-1.21,-1.19]
      # elbow_joint: [-2.4,-2.2]
      # wrist_1_joint: [-1.3,-1.1]
      # wrist_2_joint: [1.5,1.6]
      # wrist_3_joint: [2.7,2.9]
      shoulder_pan_joint: [-0.1,0.1]
      shoulder_lift_joint: [-1.6,-1.5]
      elbow_joint: [-0.1,0.1]
      wrist_1_joint: [-1.6,-1.5]
      wrist_2_joint: [-0.1,0.1]
      wrist_3_joint: [-0.1,0.1]
